### ISSUE TITLE
xcmsRaw returns NULL for mslevel = 1

### ISSUE THREAD

**Issue by jotsetung** [sneumann/xcms#101]
`xcmsRaw` has parameters `includeMSn` and `mslevel`, `includeMSn` is used in the data import function, but the `mslevel` is used at the very end of the function:

```{r}
    if (!missing(mslevel) & !is.null(mslevel)) {
        object <- msn2ms(object)
        object <- split(object, f=object@msnLevel==mslevel)$"TRUE"
        ## fix xcmsRaw metadata, or always calculate later than here ?
    }
```

I don't really get the sense of these lines, since it is also `TRUE` if `mslevel = 1` is specified - which makes no sense here.

**Comment by jotsetung** [comment]
Tested also on the `RMassBank` and `mtbls2` packages.

**Comment by jotsetung** [comment]
Fixed in `xcms`, wait for merge into `devel`.

**Comment by sneumann** [comment]
Fixed with #d559e6306b40af9a7b099a9359a003e85540feb7.
RMassBank is happy now:
http://bioconductor.org/checkResults/devel/bioc-LATEST/RMassBank/
and so is mtbls2:
http://www.bioconductor.org/checkResults/3.5/data-experiment-LATEST/mtbls2/
